rosdep/python.yaml

```diff
@@ -10075,6 +10079,7 @@ python3-singleton-pattern-decorator-pip:
     pip:
       packages: [singleton-pattern-decorator]
 python3-sip:
+  alpine: [py3-sip]
```

Comment:
I think this should be `py3-sip4-dev` on >3.14 like
https://github.com/alpine-ros/rosdistro/blob/201345f7afe8e5a994b1048cd70af0d8b69bf4bd/rosdep/python.yaml#L9442-L9447

Official Alpine repository upgraded py3-sip to v6 and ROS packages depends on v4.

Reply:
Done https://github.com/alpine-ros/rosdistro/pull/211/commits/66bb16faca8e079c20e4c42dc72a4595ff578afe